routes/api/users.js

```diff
@@ -1,12 +1,43 @@
 const express = require('express');
 const bcrypt = require('bcryptjs');
 const jwt = require('jsonwebtoken');
+const gravatar = require('gravatar');
 const User = require('../../models/user');
 const { validateSignup, validateLogin } = require('../../validation/users');
 const auth = require('../../middleware/auth');
+const multer = require('multer');
+const jimp = require('jimp');
+const { v4: uuidv4 } = require('uuid');
+const path = require('path');
+const fs = require('fs/promises');
 
 const router = express.Router();
 
+const storage = multer.diskStorage({
```

Comment:
To jest trochę duplikat innego pliku upload.js. Nie widze potrzeby zeby miec dwa wiec moze zrob jedna wersje. Ta opcja ze wszystko w middleware upload.js jest lepsza.

Reply:
Tak zrobiłam, przeniosłam wszystko do upload.js